include/dcsam/DCSAM.h

```diff
@@ -217,6 +217,7 @@ class DCSAM {
   // NOTE: will be needed for more sophisticated iSAM2 bookkeeping (rather than
   // naively iterating over all factors as we do now)
   std::vector<DCContinuousFactor::shared_ptr> dcContinuousFactors_;
+  gtsam::FastVector<gtsam::DiscreteFactor::shared_ptr> dcDiscreteFactors_;
   std::map<size_t, gtsam::FactorIndex> dcIdxToFactor_;
```

Comment:
Maybe this can be removed if unneeded?

Reply:
Assuming you meant line 221 but yeah it is not needed, thanks!